Makefile

```diff
@@ -1,7 +1,7 @@
 # A Self-Documenting Makefile: http://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
 
 GOLANGCI_VERSION=v1.61.0
-COVERAGE=coverage.out
+COVERAGE?=coverage.out
```

Comment:
Coverage reported in comments on this PR is not working because this change is not in master.

I've tried it locally and changed this line in master and everything should work perfectly